libs/providers/flagd-web/src/e2e/constants.ts

```diff
@@ -0,0 +1,3 @@
+export const FLAGD_NAME = 'flagd-web';
```

Comment:
TBH we should maybe export this from the provider itself (the provider has a name).

Reply:
i will leave this for now, and tackle it at a later stage